src/aac/templates/genplug/plugin.py.jinja2

```diff
@@ -61,4 +64,3 @@ PLUGIN_EXTENSION_YAML = """
 {%- endfor %}
 """
 {%- endif %}
-
```

Comment:
Double check this, the generated python file might be missing its final blank line and might fail flake linting.

Reply:
I didn't mean to delete that newline. Good catch.

Reply:
Resolved in 91dc894